scripts/ability_map_graph.gd

```diff
@@ -1,8 +1,8 @@
 extends GraphEdit
 
-const db_path = "res://assets/dao.sqlite"
-const unselected_color = Color.WHITE
-const selected_color = Color.GREEN
+const db_path: String = "res://assets/dao.sqlite"
+const unselected_color: Color = Color.WHITE
+const selected_color: Color = Color.GREEN
```

Comment:
Normally, I would agree with you here, but Godot/GDScript is bad about offering type hints when you go to act on the items later. For the Color declarations, that is fine since we never do anything besides provide it as an argument to another function call. 

But even the String type doesn't give you hints about what methods you can run against it unless you declare the type statically at time of definition. Since we're still learning the engine and GDScript isn't a language we are overly familiar with, better early error detection and type hints seem useful. 

This is the same with the Dictionary types and things like the FlatStyleBox. If we don't statically type it, we have a lot more of a difficult time finding appropriate functions on the fly instead of going to the documentation for every new use of an object.

Reply:
At minimum, I think all our custom-defined methods should provide static types for their parameters. But I do think that some of the other spots I added static types to make type hints better. There may be areas where it does not actually make a difference. 🤷

Reply:
According to this you can do := and the type will be inferred.  Then the variable will be treated as statically typed.  
Does that seem alright to you? 
https://godotengine.org/article/optional-typing-gdscript/#syntax

Reply:
ooh that can probably take care of a lot of it. We'll do that for now. I'll make a new PR and ask for your review after